api.md

```diff
@@ -580,11 +580,33 @@ the same semantics as the `show_chunks` [function](#show_chunks).
 
 |Name|Description|
 |---|---|
-| `table_name` | Hypertable name from which to drop chunks. If not supplied, all hypertables are affected.
+| `table_name` | Hypertable name or continuous aggregate view from which to drop chunks. This is optional, but see below.
```

Comment:
@mkindahl Could you remove this line. Lets not draw attention to this. Even though it often works , it causes weird unintelligible errors when there are tables with different datatypes for time column.

Reply:
Yes, and users will run into them. Isn't it better to explain what is going on?

Reply:
I strongly think it is better to point them to correct API usage. We don't document it as optional even though the API might work correctly in some cases. But in any non-trivial setup, users will run into issues. (Don't think we actually have many people using this with optional table_name, otherwise would have had bugs reported).

Reply:
I think it is better to give users something to search for and find a solution, but I agree that we cannot make the option required (because of backwards compatibility) and it does not make sense to support it going forwards, so we should softly deprecate it.

I added a section under `drop_chunks` with known issues and outlined what does not work and what the solution is.